apps/api/v2/src/ee/bookings/2024-08-13/inputs/booking-references-filter.input.ts

```diff
@@ -1,14 +1,14 @@
 import { ApiProperty } from "@nestjs/swagger";
-import { IsEnum, IsOptional } from "class-validator";
+import { IsIn, IsOptional } from "class-validator";
 
-export enum BookingReferenceType {
-  GOOGLE_CALENDAR = "google_calendar",
-  OFFICE365_CALENDAR = "office365_calendar",
-  DAILY_VIDEO = "daily_video",
-  GOOGLE_VIDEO = "google_video",
-  OFFICE365_VIDEO = "office365_video",
-  ZOOM_VIDEO = "zoom_video",
-}
+export const BookingReferenceType = [
```

Comment:
Hmm, maybe i would call this `BookingReferences` instead and then define `BookingReferenceType = (typeof BookingReferences)[number]` because now `BookingReferenceType` is an array not individual type.